manuscript/teaching-reproducibility.Rmd

```diff
@@ -623,17 +638,17 @@ because there has been a notable shift in that domain
 and there is now a more effective package to teach,
 e.g. switching to teaching a declarative visualization package in place of an imperative one.
```

Comment:
Should we reduce discussion (or some of the details of this discussion) of changes to the fast moving tech stack and how it impacts live demos? In particular, the discussion of the pipe and viz libraries seems to drift a bit too far at the end of this section from topic of our paper. Thoughts?

Reply:
I added this to try to address the reviewer comments about specific, but I also felt that it was getting a bit off-topic. I removed the R pipe and viz course details. I also changed a bit to contrast the github change with fundamental concepts rather than with novice programming as I think this is more of the comparison we want to make rather than contrast git/github as a less stable tool.